### PR TITLE
[#159] [#154] Dex SSO auth

### PR DESCRIPTION
**Pull request by MaksymBilenko**
This PR solves #159 #154

### CODE REVIEW

**Comment by alexander-semenets** [comment]
ok to test

File: deploy/helms/oauth2-proxy/values.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  cookie_domain: .chart-example.local
+  cookie_secure: true
+  cookie_expire: 24h
+  email_domains: "*"
+  set_authorization_header: true
+  skip_provider_button: true
+  scope: openid profile email offline_access groups
+  http_address: 0.0.0.0:4180
+  #cookie_secret: #Generate random secret if nil
+  client_secret: ZXhhbXBsZS1hcHAtc2VjcmV0
```
[the remark this answers is not in the document]
Comment (MaksymBilenko) [reply deploy/helms/oauth2-proxy/values.yaml]: This is default Dex secret that placed on dex documentation. It doesnt work and it's there as a placeholder

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Comment by MaksymBilenko** [comment]
ok to test

**Pull request closed without merging**